src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalibratorDataflowUtils.java

```diff
@@ -313,12 +353,22 @@ public void finishBundle(DoFn<KV<String, CoGbkResult>, RecalibrationTables>.Cont
             .named("finalizeRecalTables")
             .of(new DoFn<RecalibrationTables, RecalibrationTables>() {
```

Comment:
OK, I've added one. To be maximally flexible I allow you to specify a different class and operation name. I also chose not to override processElement yet, though that's a matter of choice - perhaps later we may wish to override it to count the number of calls, perhaps time them automatically. Of course that may not make sense if processElement is very fast.